lib/rules/disallow-space-between-arguments.js

```diff
@@ -46,20 +46,13 @@ module.exports.prototype = {
     },
 
     check: function(file, errors) {
-
-        var tokens = file.getTokens();
-
         file.iterateNodesByType(['CallExpression'], function(node) {
-
             node.arguments.forEach(function(param, i) {
-
-                var punctuatorToken = file.getTokenByRangeStart(param.range[1]);
-
+                var punctuatorToken = file.getPrevToken(file.getFirstNodeToken(param));
```

Comment:
You using here `punctuatorToken = getPrevToken` and later `nextToken: file.getNextToken(punctuatorToken)`.

So `file.getFirstNodeToken` is already a `nextToken` for `punctuatorToken`.

You can get rid of unnecessary `getNetToken` call by storing `firstNodeToken`.